siege_utilities/geo/spatial_data.py

```
@@ -403,6 +449,58 @@ def get_year_specific_url_patterns(self, year: int) -> Dict[str, Any]:
+        if year == 2020:
+            suffix = 'vtd20'
+            pl_dir = 'TIGER2020PL'
+        elif year == 2010:
+            suffix = 'vtd10'
+            pl_dir = 'TIGER2010PL'
+        else:
+            raise BoundaryDiscoveryError(
+                f"VTD PL boundaries are only available for decennial years "
+                f"(2020, 2010).  Requested year: {year}.",
+                context={"year": year, "boundary_type": "vtd"},
+            )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)